### ISSUE TITLE
[Feature request] Changing the position of columns on the grid

### ISSUE THREAD

**Issue by mansurdegirmenci** [vitmalina/w2ui#2083]
I know you have this feature before. my suggestion can be done by making "show//hide columns" move in place. I think it's simpler.

![image](https://user-images.githubusercontent.com/38755096/130361558-18f92269-25a9-41cc-a836-e1a012f96694.png)

We can move the columns on the "grid" by dragging the row up or down.

**Comment by mrg2001** [comment]
Yes, it is in the plans to remove this button and open this menu when right clicked on any column

**Comment by mansurdegirmenci** [comment]
no no what i want to say is; I'm talking about the order of the columns on the grid.

**Comment by mrg2001** [comment]
I see. Will consider

**Comment by mpf82** [comment]
Re-opening this feature request. I will talk to Vitali if somewhere along the w2ui 2.0 we will replace this drop-down list with a more complex dialog, e.g. similar to this one that we're using in our projects.

You could then reorder column with drag&drop as indicated in the screenshot or use the buttons in the toolbar to move columns to top/bottom or up/down.

![2021-08-30 13_20_07-Window](https://user-images.githubusercontent.com/10152404/131332003-1037cc0c-8a50-45fb-8f8b-397285948fb4.png)

This could also let you select the column renderer on the fly:

![2021-08-30 13_25_36-Window](https://user-images.githubusercontent.com/10152404/131332340-3addbdfd-efea-416f-b7ca-5d72a8039d9d.png)
